package.json

```diff
@@ -23,6 +23,7 @@
     "botbuilder": "^3.2.3",
     "passport": "^0.3.2",
     "passport-azure-ad": "2.0.0",
+    "querystring": "^0.2.0",
```

Comment:
kind of confused; are you using the builtin `querystring` module in your code? (https://nodejs.org/api/querystring.html)